src/Tribe/Template.php

```diff
@@ -73,6 +73,15 @@ class Tribe__Template {
 	 */
 	protected $template_folder_lookup = false;
 
+	/**
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 0b89ca8c812285cf21d1d2f58c6ac383</sub>